Description: Skip ECL interrupt doctest
 ECL 24.5 aborts when handling interrupts if the first environment is
 not page-aligned (ECL issue #828).
Author: Maximiliano Curia <maxy@debian.org>

Index: sagemath/src/sage/libs/ecl.pyx
===================================================================
--- sagemath.orig/src/sage/libs/ecl.pyx	2026-09-23 22:00:16.867975775 +0200
+++ sagemath/src/sage/libs/ecl.pyx	2026-10-07 11:29:06.428121984 +0200
@@ -304,8 +304,8 @@
         sage: ecl_eval("(setf i 0)")
         <ECL: 0>
         sage: inf_loop = ecl_eval("(defun infinite() (loop (incf i)))")
-        sage: interrupt_after_delay(1000)
-        sage: inf_loop()
+        sage: interrupt_after_delay(1000)  # not tested
+        sage: inf_loop()  # not tested
         Traceback (most recent call last):
         ...
         KeyboardInterrupt: ECL says: Console interrupt.
